src/Microsoft.Restier.Security/ApiConfigurationExtensions.cs

```diff
@@ -32,8 +32,8 @@ public static class ApiConfigurationExtensions
             this ApiBuilder builder)
         {
             Ensure.NotNull(builder, "builder");
-            builder.AddHookHandler<IQueryExpressionInspector>(RoleBasedAuthorization.Default);
-            builder.AddHookHandler<IQueryExpressionExpander>(RoleBasedAuthorization.Default);
+            builder.CutoffPrevious<IQueryExpressionInspector, RoleBasedAuthorization>();
+            builder.ChainPrevious<IQueryExpressionExpander, RoleBasedAuthorization>();
```

Comment:
Since RoleBasedAuthorization has no state, this change won't break anything. But still I'd like to split the 2 services to make it clearer and avoid any inadvertent change to break the semantic.